test/opt/spread_volatile_semantics_test.cpp

```diff
@@ -1133,6 +1134,160 @@ OpFunctionEnd
   EXPECT_EQ(status, Pass::Status::SuccessWithoutChange);
 }
 
+TEST_F(VolatileSpreadTest, NoInlinedfuncCalls) {
```

Comment:
This is a very large module for something that I believe could be much smaller.  Could you reduce the test case?